cmd/install.go

```diff
@@ -87,6 +88,7 @@ func newInstallCmd() *installCmd {
 				Hash:        fmt.Sprintf("%x", pResult.Hash.Sum(nil)),
 				URL:         u,
 				Provider:    p.GetID(),
+				LatestURL:   root.opts.latestURL,
```

Comment:
Same as other comments, having this in the top level of the config seems weird. I was thinking that we could maybe extend the `provider` key we already have and make it `map` instead of `string`. That way, we can save provider specific configurations in the config. 

We could add a new `GetConfig` methgod to the provider interface for this.